extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -157,6 +211,10 @@ export class IframeTransport {
     }
     ampDoc.body.removeChild(frameData.frame);
     delete IframeTransport.crossDomainIframes_[type];
+    if (this.longTaskObserver_) {
+      this.longTaskObserver_.disconnect();
+      delete this.longTaskObserver_;
```

Comment:
Let's set to null to be consistent with constructor

Reply:
Done